vibes/vibed-ui/src/ui/Font.cpp

```
@@ -0,0 +1,33 @@
+// MIT License
+// Copyright (c) 2026 Palaash
+
+#include "Font.h"
+
+namespace ui {
+
+Font::Font() : fontName("Sans"), fontSize(12) {}
+
+Font::Font(const std::string& name, int size)
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: fontName

(cppcoreguidelines-pro-type-member-init)